server/athenian/api/models/state/versions/79c33483be7d_use_jsonb.py

```diff
@@ -0,0 +1,120 @@
+"""Use JSONB
+
+Revision ID: 79c33483be7d
+Revises: 9172948308c9
+Create Date: 2022-07-28 09:17:55.121271+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "79c33483be7d"
+down_revision = "9172948308c9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.batch_alter_table("account_features") as bop:
```

Comment:
Only if the dialect is `postgresql`, otherwise, return immediately

Reply:
this was a no-op with sqlite,anyway added